### ISSUE TITLE
[stable/openldap] Helm3 fails with "ClusterIP is immutable" error

### ISSUE THREAD

**Issue by Morishiri** [helm/charts#20904]
**Describe the bug**
When updating the release with helm3 it throws an error about ClusterIP not being mutable.

**Version of Helm and Kubernetes**:
```
$  helm version  
version.BuildInfo{Version:"v3.1.0", GitCommit:"b29d20baf09943e134c2fa5e1e1cab3bf93315fa", GitTreeState:"clean", GoVersion:"go1.13.7"}
```
```
$  kubectl version           
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.2+k3s1", GitCommit:"cdab19b09a84389ffbf57bebd33871c60b1d6b28", GitTreeState:"clean", BuildDate:"2020-01-27T18:09:26Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"linux/amd64"}
```
**Which chart**:
`stable/openldap`

**What happened**:
When updating the release with helm3 it throws:
```
Error: UPGRADE FAILED: cannot patch "ldap-openldap" with kind Service: Service "ldap-openldap" is invalid: spec.clusterIP: Invalid value: "": field is immutable: cannot patch "ldap-openldap" with kind Service: Service "ldap-openldap" is invalid: spec.clusterIP: Invalid value: "": field is immutable
```

**What you expected to happen**:
Upgrade goes without issues because the field was not actually mutated

**How to reproduce it** (as minimally and precisely as possible):
`helm install ldap stable/openldap`

`helm upgrade ldap stable/openldap --set replicaCount=2`

**Anything else we need to know**:
I will provide the PR with solution.

**Comment by Morishiri** [comment]
remove lifecycle stale

**Comment by Morishiri** [comment]
remove lifecycle stale

**Comment by chirangaalwis** [comment]
I am experiencing this error as well. I am using Helm version `3.1.2` with GKE based Kubernetes Server version: 1.14+, Git Version: `v1.14.10-gke.27`.

**Comment by kyounger** [comment]
Bump. Issue for me, too.

**Comment by kyounger** [comment]
I tested by pulling the repo, checking out the PR, and using the chart locally to install. Using helm 3.2.0.

**Comment by chirangaalwis** [comment]
> I tested by pulling the repo, checking out the PR, and using the chart locally to install. Using helm 3.2.0. Fix seems to work fine.

@kyounger it seems the chart source in this repository has not been updated for quite a while (at least 5 months as of now). The relevant [chart](https://hub.helm.sh/charts/stable/openldap) of version `1.2.3` had been pushed during this month to the chart repository.

Shouldn't this change be merged if appropriate and made available in the chart artifact available at the stable chart repository?

**Comment by chirangaalwis** [comment]
It seems that this has been fixed in version `1.2.4` and is available at the [Helm Hub](https://hub.helm.sh/charts/stable/openldap/1.2.4).

LGTM!!! Thanks @Morishiri
